support-frontend/assets/pages/weekly-subscription-landing/components/content/Benefits.jsx

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import React from 'react';
+import List from 'components/list/List';
+import BenefitsContainer from './BenefitsContainer';
+import BenefitsHeading from './BenefitsHeading';
+
+function Benefits() {
+  return (
+    <BenefitsContainer
+      sections={[
+        {
+          id: 'benefits',
+          content: (
+            <>
```

Comment:
Oooh fragments...! Are they working for you?

Reply:
I've not had any issues! I really like not having to introduce extra elements to the DOM just to group things together, especially when passing children like this. Make it easier to reason about the layout too.